tracing-subscriber/src/registry/sharded.rs

```diff
@@ -0,0 +1,253 @@
+use sharded_slab::{Guard, Slab};
+
+use super::stack::SpanStack;
+use crate::{
+    registry::{
+        extensions::{Extensions, ExtensionsInner, ExtensionsMut},
+        LookupSpan, SpanData,
+    },
+    sync::RwLock,
+};
+use std::{
+    cell::RefCell,
+    sync::atomic::{fence, AtomicUsize, Ordering},
+};
+use tracing_core::{
+    dispatcher,
+    field::FieldSet,
+    span::{self, Current, Id},
+    Event, Interest, Metadata, Subscriber,
+};
+
+/// A shared, reusable store for spans.
+///
+/// This registry is implemented using a [lock-free sharded slab][1], and is
+/// highly optimized for concurrent access.
+///
+/// [1]: https://docs.rs/crate/sharded-slab/0.0.5
```

Comment:
Is there any particular reason to link to a specific version here?

Reply:
no, we probably shouldn't refer to a version by name...i think this is just because i copied and pasted a URL from firefox :P
```suggestion
/// [1]: https://docs.rs/crate/sharded-slab
```